core/schema/container.go

```diff
@@ -524,8 +524,8 @@ func (s *containerSchema) Install() {
 
 		dagql.Func("export", s.export).
 			View(AllVersion).
-			Impure("Writes to the local host.").
-			Doc(`Writes the container as an OCI tarball to the destination file path on the host.`,
+			Impure("Writes to the current runtime container.").
```

Comment:
This isn't entirely correct. This is only true with modules, but you can still run dagger pipelines without modules, in which case it does write to your host.

Reply:
It writes to the filesystem where the client call is made from. As it turns out, since modules run in a container, it's going to be exported to that container. If you run the pipeline directly from your host (i.e., `dagger run`) then that's where it will be exported to.

I'd prefer to avoid making the distinction here of running in a module vs non-module, as being the two options. On one hand, "host" can easily lead you astray. On the other hand, "runtime container" is too specific to this use case. Additionally "non-module" has implications with terminology that aren't cemented yet.

Best if there's a general way to phrase this, in line with "writes to the filesystem where the API call is made from" (i.e., avoiding the usual connotation to the word "host", but still general enough that it can match a container, virtual machine, etc...), but add a cautionary _note_ in the longer description with something like "when running in a module, it's written to the runtime container setup by the SDK".